.github/workflows/crowdin.yml

```diff
@@ -13,31 +13,55 @@ concurrency:
 
 jobs:
   synchronize-with-crowdin:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
+    container: swift:5.10.1
 
-    steps:      
+    steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Crowdin action
         uses: crowdin/github-action@v2
         with:
           # For more info: https://github.com/crowdin/github-action/blob/master/action.yml
-          project_id: ${{ secrets.CROWDIN_PROJECT_ID }}          
+          project_id: ${{ secrets.CROWDIN_PROJECT_ID }}
           token: ${{ secrets.CROWDIN_API_TOKEN }}
           config: crowdin.yml
 
           upload_sources: true
           download_translations: true
           upload_translations: false
-
-          create_pull_request: true
+          create_pull_request: false
+          push_translations: true
```

Comment:
why do we need this ?

Reply:
The crowdin action populates the string catalog file with many unused translations. Since then it detects a diff it would create a PR.
However, we instead want to remove the unused translates and then check again, if there is still a diff.

Reply:
is that a bug at the end of the crowdin action, can we open an issue or fix it there ?

Reply:
We should definitely reach out to them.